### DISCUSSION TITLE
Object.defineProperty called on non-object

### DISCUSSION THREAD

**Discussion by caiqueacct** [feathersjs/feathers#3289] [Q&A]
I created a MySQL table where the ID is a varchar(50) and there is a before insert trigger that update the field with UUID() data.
The POST works correctly inserting the data into the table but the return to the client is an error message:

{
    "message": "Object.defineProperty called on non-object"
}

Any ideas?

Version: 5.0.9

Thanks.

**Comment by daffl** [comment]
There should be more information available somewhere, like the stack trace logged in the console.

**Comment by caiqueacct** [comment]
I created a hook and forced to update the id field with context.arguments[0].id = uuidv4()
